src/taipy/gui/gui.py

```diff
@@ -206,6 +208,7 @@ class Gui:
 
     __RE_HTML = re.compile(r"(.*?)\.html")
```

Comment:
In those three regexps, I suspect you should have an additional '$' at the end of the regexp.
Otherwise these will match "myfile.md.text", "file.pyi", and others...

Reply:
thanks!